doc/source/whatsnew/v0.18.0.txt

```diff
@@ -1117,3 +1117,4 @@ Bug Fixes
 - Bug in ``DataFrame.apply`` in which reduction was not being prevented for cases in which ``dtype`` was not a numpy dtype (:issue:`12244`)
 - Bug when initializing categorical series with a scalar value. (:issue:`12336`)
 - Bug when specifying a UTC ``DatetimeIndex`` by setting ``utc=True`` in ``.to_datetime`` (:issue:11934)
+- Bug in ``IndexEngine.get_loc`` in which an empty ``TypeError`` was being thrown instead of one with an error message (:issue:12218)
```

Comment:
just say that you added a meaningful error message when multi-axis indexing with an invalid value.

these whatsnotes are for the average user. Those who want/need more detail can simply click on the issue itself. But for a quick glance, just need to highlite terms that most users would know.